src/client/Microsoft.Identity.Client/AppConfig/AssertionRequestOptions.cs

```diff
@@ -30,5 +31,11 @@ public class AssertionRequestOptions {
         /// Claims to be included in the client assertion
         /// </summary>
         public string Claims { get; set; }
+
+        /// <summary>
+        /// Capabilities that the client application declares in the assertion.
+        /// The service may require or interpret these capabilities in advanced scenarios.
+        /// </summary>
+        public IEnumerable<string> ClientCapabilities { get; set; }
```

Comment:
Does `WithClientCapabilities` have any impact on this? Is `WithClientCapabilities` available on the ManagedIdentityApp?

Reply:
Yes, there is `WithClientCapabilities` on MIA